doodle-formats/src/format/jpeg.rs

```diff
@@ -17,7 +17,7 @@ pub fn main(module: &mut FormatModule, base: &BaseModule, tiff: &FormatRef) -> F
             ("length", base.u16be()),
             (
                 "data",
-                Format::Slice(sub(var("length"), Expr::U16(2)), Box::new(data)),
+                Format::Slice(Box::new(sub(var("length"), Expr::U16(2))), Box::new(data)),
```

Comment:
Similarly a slice helper if we don't already have one.

Reply:
Similarly fixing this.